.github/workflows/api-cd.yml

```
@@ -0,0 +1,87 @@
+      MYSQL_HOST: ${{ secrets.MYSQL_HOST }}
+      MYSQL_PORT: ${{ secrets.MYSQL_PORT }}
+      MYSQL_DATABASE: ${{ secrets.MYSQL_DATABASE }}
+    steps:
+      - name: 📦 Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: 🚀 Deploy
+        run: |
+          echo "$(date): 컨테이너 이미지 풀"
+          docker compose pull api
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)